examples/wallets/import_wallet/main.go

```diff
@@ -0,0 +1,168 @@
+)
+
+func main() {
+
+	// Insert the wallet mnemonic you want to import
+	mnemonic := "<your_mnemonic_here>"
+
+	// Organization ID, user ID and API private key
+	organizationId := "<orgId>"
+	userId := "<user_from_orgId>"
+	apiPrivateKey := "<private_key_here>"
+
+	// Generate a new key pair used to encrypt the export bundle
+	encryptionKey, err := encryptionkey.New(userId, organizationId)
+
+	if err != nil {
+		log.Fatal("creating encryption key: %w", err)
+	}
+
+	// API key used by the client
+	apiKey, err := apikey.FromTurnkeyPrivateKey(apiPrivateKey, apikey.SchemeP256)
+
+	if err != nil {
+		log.Fatal("creating API key: %w", err)
+	}
+
+	client, err := sdk.New(sdk.WithAPIKey(apiKey))
+
+	if err != nil {
+		log.Fatal("creating SDK client: %w", err)
+	}
+
+	signerKey, err := hexToPublicKey(encryptionkey.SignerProductionPublicKey)
+
+	if err != nil {
+		log.Fatal("failed to convert the public key")
+	}
+
+	// Get the private key
+	tkPrivateKey := encryptionKey.GetPrivateKey()
+	kemPrivateKey, err := encryptionkey.DecodeTurnkeyPrivateKey(tkPrivateKey)
+
+	if err != nil {
+		log.Fatal("failed to decode encryption private key")
+	}
+
+	// Set up enclave encrypt client
+	encryptClient, err := enclave_encrypt.NewEnclaveEncryptClientFromTargetKey(signerKey, *kemPrivateKey)
+
+	if err != nil {
+		log.Fatal("failed to setup enclave encrypt client")
+	}
+
+	// Init import activity, this produces an import bundle, containing a public key and signature.
+	// These artifacts will be used in the next step to ensure that key material is only accessible by Turnkey, and cannot be extracted by any man-in-the-middle (MITM)
+	initImportParams := wallets.NewInitImportWalletParams().WithBody(&models.InitImportWalletRequest{
+		OrganizationID: &organizationId,
+		Parameters: &models.InitImportWalletIntent{
+			UserID: &userId,
+		},
+		TimestampMs: util.RequestTimestamp(),
+		Type:        (*string)(models.ActivityTypeInitImportWallet.Pointer()),
+	})
+
+	reply, err := client.V0().Wallets.InitImportWallet(initImportParams, client.Authenticator)
+
+	if err != nil {
+		log.Fatal("init import request failed: %w", err)
+	}
+
+	importBundle := *reply.Payload.Activity.Result.InitImportWalletResult.ImportBundle
+
+	clientSendMsg, err := encryptClient.Encrypt([]byte(mnemonic), []byte(importBundle), organizationId, userId)
+
+	if err != nil {
+		log.Fatal("unable to encrypt wallet to target: %w", err)
+	}
+
+	encryptedBundle, err := json.Marshal(clientSendMsg)
+
+	if err != nil {
+		log.Fatal("failed to encrypt bundle: %w", err)
+	}
+
+	// Perform import activity
+	importParams := wallets.NewImportWalletParams().WithBody(&models.ImportWalletRequest{
+		OrganizationID: &organizationId,
+		Parameters: &models.ImportWalletIntent{
+			UserID: &userId,
+			Accounts: []*models.WalletAccountParams{
+				{
+					AddressFormat: models.AddressFormatSolana.Pointer(),
+					Curve:         models.CurveEd25519.Pointer(),
+					Path:          util.StringPointer("m/44'/60'/1'/0/0"),
```

Comment:
could be worthwhile pointing to other common paths https://docs.turnkey.com/concepts/wallets#hd-wallet-default-paths